frontend/packages/pipelines-plugin/src/models/pipelines.ts

```diff
@@ -205,3 +205,22 @@ export const EventListenerModel: K8sKind = {
   badge: BadgeType.TECH,
   color,
 };
+
+export const RepositoryModel: K8sKind = {
+  apiGroup: 'pipelinesascode.tekton.dev',
+  apiVersion: 'v1alpha1',
+  label: 'Repository',
+  // t('pipelines-plugin~Repository')
+  labelKey: 'pipelines-plugin~Repository',
+  // t('pipelines-plugin~Repository')
+  labelPluralKey: 'pipelines-plugin~Repositories',
+  plural: 'repositories',
+  abbr: 'R',
+  namespaced: true,
+  kind: 'Repository',
+  id: 'repository',
+  labelPlural: 'reposities',
```

Comment:
Once you Other PR goes in, I am assuming these old changes will be removed in this pr

Reply:
yes